src/components/fabToolbar/fabToolbar.js

```diff
@@ -0,0 +1,134 @@
+   *
+   * <hljs lang="html">
+   * <md-fab-toolbar>
+   *   <md-fab-trigger>
+   *     <md-button><md-icon icon="/img/icons/plus.svg"></md-icon></md-button>
+   *   </md-fab-trigger>
+   *
+   *   <md-fab-actions>
+   *     <md-button><md-icon icon="/img/icons/user.svg"></md-icon></md-button>
+   *     <md-button><md-icon icon="/img/icons/group.svg"></md-icon></md-button>
+   *   </md-fab-actions>
+   * </md-fab-toolbar>
+   * </hljs>
+   *
+   * @param {expression=} md-open Programmatically control whether or not the toolbar is visible.
+   */
+  function MdFabToolbarDirective($timeout) {
+    return {
+      restrict: 'E',
+      transclude: true,
+      template: [
+        '<div class="md-fab-toolbar-wrapper">',
+        '  <div class="md-fab-toolbar-background"></div>',
+        '  <div ng-transclude></div>',
+        '</div>'
+      ].join('\n'),
+
+      scope: {
+        isOpen: '=?mdOpen',
+      },
+
+      controller: controller,
+
+      link: link
+    };
+
+    function controller($scope, $element) {
+      var vm = this;
+
+      // Set the default to be closed
+      if (typeof $scope.isOpen === 'undefined') {
+        $scope.isOpen = false;
+      }
+
+      vm.focus = function() {
+        // TODO: Why does this need to be inside a $timeout???
+        $timeout(function() {
+          $scope.isOpen = true;
+        });
+      };
+
+      vm.blur = function() {
+        $timeout(function() {
+          $scope.isOpen = false;
+        });
+      };
+    }
+
+    function link($scope, $element, $attriubutes, ctrl) {
+      var i, focusable, child, children = [], types = ['button', 'a'];
+
+      children = $element.find("button");
+      //children = children.concat($element.find("a"));
+
+      // Add some focus/blur events to whatever (post-compiled) elements the user added in the toolbar
+      for (i = 0; i < children.length; i++) {
+        focusable = children.eq(i);
+
+        focusable.on('focus', ctrl.focus);
+        focusable.on('blur', ctrl.blur);
+      }
+
+      $element.on('mouseenter', ctrl.focus);
+      $element.on('mouseleave', ctrl.blur);
+
+      // Don't allow focus on the trigger
+      $element.find('md-fab-trigger').find('button').attr('tabindex', '-1');
+
+      // Watch for changes to md-open
+      $scope.$watch('isOpen', function(isOpen) {
+        // Grab the elements
+        var backgroundElement = $element.children().eq(0).children().eq(0)[0];
+        var triggerElement = $element.children().eq(0).children().eq(1).children().eq(0)[0];
+
+        // If we find both elements, use them to position the new backgroundt co
+        if (triggerElement && backgroundElement) {
+          // Get our variables
+          var width = $element[0].offsetWidth;
+          var height = $element[0].offsetHeight;
+
+          // Make a square
+          var scaleWidth = width * 2;
+          var scaleHeight = width * 2;
+
+          // Set the positioning and border
+          var borderRadius = width;
```

Comment:
Since you only use it in 
```js 
backgroundElement.style.borderRadius = borderRadius + 'px';
```
I was suggesting just:
```js 
backgroundElement.style.borderRadius = width + 'px';
```